doc/database.md

```diff
@@ -86,3 +88,34 @@ Some reasons for a transcoder not claiming a receipt:
 * The transcoder did not submit a claim within 256 blocks of being assigned a job.
 * There was an error while processing the segment.
 * [Claim interval policy](https://github.com/livepeer/go-livepeer/issues/371) (unimplemented)
+
+## Table `broadcasts`
+
+**Broadcaster only.** Tracks jobs that broadcasters have created. This is used for job reuse, among other things.
+
+Column | Type | Description
+---|---|---
+id | INTEGER PRIMARY KEY | The ID of the on-chain job
+recordedAt | STRING DEFAULT CURRENT_TIMESTAMP | Time this row was inserted, *after* the job transaction has been confirmed on chain.
+streamID | STRING | Stream identifier, composed of `<VideoID><Rendition>`.
+segmentPrice | INTEGER | The cost of transcoding each segment, in wei.
+segmentCount | INTEGER DEFAULT 0 | The number of segments this job has produced.
+transcodeOptions | STRING | [Encoded version of the rendition strings](https://github.com/livepeer/go-livepeer/blob/fac0393950e821f16555a17eb460b8abe74f6ada/common/videoprofile_ids.go#L5-L15).
+broadcaster | STRING | Broadaster's Eth address. Should be your node's address.
+transcoder | STRING | Transcoder's Eth address.
+startBlock | INTEGER | Block height from when the job was submitted on-chain.
+endBlock | INTEGER | Block height at which the job expires.
+stopReason | STRING DEFAULT NULL | Reason for stopping a job on the broadcaster. This may be before `stopBlock` is reached.
+stoppedAt | STRING DEFAULT NULL | Time which the job was stopped on the broadcaster.
+
+## Table `unbondingLocks`
+
+**All Nodes** Tracks unbonding in order to support partial unbonding.
+
+Column | Type | Description
+---|---|---
+id | INTEGER NOT NULL | The ID of the on-chain unbonding lock
+delegator | STRING | The owner of the unbonding lock. Should be your node's address.
+amount | TEXT | Amount of token to be unbonded.
+withdrawRound | int64 | Round at which unbonding period is over and tokens can be withdrawn.
+usedBlock | int64 | Block at which the token held by this lock were withdrawn.
```

Comment:
At the moment, `usedBlock` can actually represent:

- Block at which the unbonding lock is used to withdraw the locked amount of tokens
- Block at which the unbonding lock is used to rebond the locked amount of tokens

Reply:
Rewrote to "Block at which the unbonding lock is used to withdraw or rebond tokens."